### PR TITLE
yashim/feat: updated cfd warnings + cookie notice design

### PR DESCRIPTION
**Pull request by yashim-deriv**
Co-authored-by: Yashim Wong <yashimwong@gmail.com>

Changes:
- CFD Warning wrapper moved to bottom
- Changed cookie notice to a modal (in desktop) and a banner (in mobile)
- Due to this new CFD Warning placement, `LiveChat` will be positioned based on whether the cookie notice is shown or not

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/cookie-banner.js [hunk 1]
```diff
@@ -29,56 +29,111 @@ const FadeOutUp = keyframes`
...
+    @media ${device.bp1060} {
+        font-size: 14px;
+    }
+    @media ${device.tablet} {
+        font-size: 12px;
+        margin-bottom: 8px;
+    }
+    @media (max-width: 512px) {
+        margin-bottom: unset;
+    }
```
Comment (sean-binary) [inline src/components/custom/cookie-banner.js:98]: What's the intention for this yeah?

File: src/components/custom/cookie-banner.js [hunk 2]
```diff
@@ -29,56 +29,111 @@ const FadeOutUp = keyframes`
...
+    @media (max-width: 1269px) {
+        bottom: -6rem;
+    }
     @media ${device.tabletL} {
-        height: auto;
+        bottom: -4rem;
+    }
+    @media ${device.tablet} {
+        position: fixed;
```
Comment (sean-binary) [inline src/components/custom/cookie-banner.js:57]:
```suggestion
```

File: src/components/custom/cookie-banner.js [hunk 3]
```diff
@@ -29,56 +29,111 @@ const FadeOutUp = keyframes`
...
+    }
+`
+
+const StyledText = styled(Text)`
+    font-size: 14px;
+    margin-bottom: 16px;
+
+    @media ${device.bp1060} {
+        font-size: 14px;
+    }
```
Comment (sean-binary) [inline src/components/custom/cookie-banner.js:91]: I don't think this is needed.

File: src/components/custom/cookie-banner.js [hunk 4]
```diff
@@ -29,56 +29,111 @@ const FadeOutUp = keyframes`
...
+        margin-bottom: 8px;
+    }
+`
+
+const LinkText = styled(LocalizedLinkText)`
+    font-size: 14px;
+
+    @media ${device.bp1060} {
+        font-size: 14px;
+    }
```
Comment (sean-binary) [inline src/components/custom/cookie-banner.js:110]: Same here.

File: src/components/elements/off-canvas-menu.js [hunk 5]
```diff
@@ -55,16 +55,15 @@ const OffCanvasMenu = styled.section`
     ${({ is_canvas_menu_open }) => is_canvas_menu_open && 'transform: translateX(254px)'};
 
     @media ${device.tabletS} {
-        top: ${(props) =>
-            props.is_eu_country ? `${cfd_warning_height.tablet + 7.2}rem` : `7.2rem`};
+        top: 7.2rem;
     }
 `
```
Comment (sean-binary) [inline src/components/elements/off-canvas-menu.js:60]: Can remove this section, no longer needed.

File: src/components/elements/off-canvas-menu.js [hunk 6]
```diff
@@ -55,16 +55,15 @@ const OffCanvasMenu = styled.section`
...
 
 const OffCanvasMenuSecondary = styled(OffCanvasMenu)`
-    top: ${(props) => (props.is_eu_country ? `${cfd_warning_height.desktop + 10}rem` : `10rem`)};
+    top: 10rem;
 
     @media ${device.tabletS} {
-        top: ${(props) => (props.is_eu_country ? `${cfd_warning_height.tablet + 10}rem` : `10rem`)};
+        top: 10rem;
     }
 `
```
Comment (sean-binary) [inline src/components/elements/off-canvas-menu.js:68]: Same here.

File: src/components/layout/layout.js [hunk 7]
```diff
@@ -66,50 +76,47 @@ const CFDText = styled(Text)`
...
 }
 
 const Main = styled.main`
     margin-top: ${(props) =>
         props.use_eu_margin
-            ? (props.margin_top && `${props.margin_top + cfd_warning_height.desktop}rem`) ||
-              `${7 + cfd_warning_height.desktop}rem`
-            : (props.margin_top && `${props.margin_top}rem`) || `7rem`};
+            ? (props.margin_top && `${props.margin_top}rem`) || '7rem'
+            : (props.margin_top && `${props.margin_top}rem`) || '7rem'};
```
Comment (sean-binary) [inline src/components/layout/layout.js:110]: What's the difference here yeah? Both conditions look the same. If so, we can just use the `&&` operator.

File: src/components/layout/layout.js [hunk 8]
```diff
@@ -66,50 +76,47 @@ const CFDText = styled(Text)`
...
 
     @media ${device.tabletS} {
         margin-top: ${(props) =>
             props.use_eu_margin
-                ? (props.margin_top && `${props.margin_top + cfd_warning_height.tablet}rem`) ||
-                  `${7 + cfd_warning_height.tablet}rem`
-                : (props.margin_top && `${props.margin_top}rem`) || `7rem`};
+                ? (props.margin_top && `${props.margin_top}rem`) || '7rem'
+                : (props.margin_top && `${props.margin_top}rem`) || '7rem'};
     }
```
Comment (sean-binary) [inline src/components/layout/layout.js:120]: I don't think this section is longer required 😀

**Review by sean-binary — COMMENTED** [review]
The rest LGTM 👍  Nice PR.

File: src/components/layout/layout.js (hunk 8 above)
Comment (yashim-deriv) [reply src/components/layout/layout.js]: 😆 yeah true

File: src/components/layout/layout.js (hunk 7 above)
Comment (yashim-deriv) [reply src/components/layout/layout.js]: Missed out on this 😅. unused rule when the warning is on top

**Review by sean-binary — DISMISSED** [review]

File: src/components/layout/livechat.js [hunk 9]
```diff
@@ -1,47 +1,72 @@
...
+                    is_eu_country={is_eu_country}
                     onClick={() => {
                         LC_API.open_chat_window()
                     }}
                     onMouseEnter={() => setLivechatHover(true)}
                     onMouseLeave={() => setLivechatHover(false)}
                 >
                     {is_livechat_hover ? (
-                        <img src={LiveChatHover} alt="livechat hover" />
+                        <img src={LiveChatHover} alt="livechat icon hover" />
```
Comment (ShuvoHabib) [inline src/components/layout/livechat.js:67]: We can remove hover from the alt

**Review by ShuvoHabib — COMMENTED** [review]
Alt tags

**Review by sean-binary — APPROVED** [review]

**Pull request merged**